docs/about/credits.rst

```diff
@@ -0,0 +1,65 @@
+*******************
+Authors and Credits
+*******************
+
+PlasmaPy Coordinating Committee
+===============================
+* Drew Leonard
+* Nick Murphy
+* Tulasi Parashar
+* Dominik Stańczak
+
+PlasmaPy Contributors
+=====================
+* Jasper Beckers
+* Ludovico Bessi
+* Sean Carroll
+* Apoorv Choubey
+* C. Clauss
```

Comment:
For some reason an extra space appears in front of this name in the docs. Not a big deal, but looks a bit off.